### PR TITLE
Bump protobuf to v21.9

### PR DESCRIPTION
**Pull request by rschu1ze**
Cf. #55963

### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by robot-clickhouse-ci-1** [comment]
*This is an automated comment for commit 50c92dfe11687e29ff424670d53eaafc5906f673 with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/56014/50c92dfe11687e29ff424670d53eaafc5906f673/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td><a href="https://github.com/ClickHouse/ClickHouse/actions/runs/6640000426">✅ success</a></td></tr>
<tbody>
</table>
</details>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/56014/50c92dfe11687e29ff424670d53eaafc5906f673/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

**Review by vitlibar — APPROVED** [review]

File: contrib/google-protobuf
[the remark this answers is not in the document]
Comment (rschu1ze) [reply contrib/google-protobuf:1]: Yes, branch [ClickHouse/v21.9](https://github.com/ClickHouse/google-protobuf/tree/ClickHouse/v21.9) in our protobuf fork.

[the remark this answers is not in the document]
Comment (vitlibar) [reply contrib/google-protobuf:1]: We could probably write that in our [.gitmodules](https://github.com/ClickHouse/ClickHouse/blob/master/.gitmodules) file:
```
path = contrib/google-protobuf
url = https://github.com/ClickHouse/google-protobuf.git
branch = master
```

[the remark this answers is not in the document]
Comment (rschu1ze) [reply contrib/google-protobuf:1]: Let's better not do that: https://github.com/ClickHouse/ClickHouse/pull/53763

**Pull request merged**